src/pages/dev/reference/glossary.mdx

```diff
@@ -0,0 +1,59 @@
+
+## Interchain Token
+An Interchain Token is an ERC-20 contract created by the Interchain Token Service or by yourself that can be transferred between blockchains. If the contract implements the `IInterchainToken` interface, it will have knowledge of its own Token Manager and will contain transfer methods directly. If it does not implement this interface, you will need to call functions on the Token Manager to transfer the token.
+
+## Interchain Token Service
+Interchain Token Service is a contract deployed by Axelar that manages the creation and management of Interchain Tokens and Token Managers.
+
+The Interchain Token Service has an address of [0xF786e21509A9D50a9aFD033B5940A2b7D872C208](https://goerli.etherscan.io/address/0xF786e21509A9D50a9aFD033B5940A2b7D872C208) on all testnet chains,
+deployed from the [v0.3.0](https://github.com/axelarnetwork/interchain-token-service/releases/tag/v0.3.0) release. Deployed addresses can be found [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json).
+
+## Lock/Release
+
+Locking and releasing in the blockchain context pertains to the temporary restriction and restoration of the movement or transfer of tokens or assets within blockchain networks. 
+
+In the Interchain Token Service, incoming token transfers will result in the local release of locked tokens. Outgoing token transfers will result in the local locking of tokens on this blockchain by the Token Manager.
+
+## Mint/Burn
+
+Minting in blockchain refers to the creation of new tokens, typically increasing the total supply. It serves various purposes, including rewards, transfers, or asset conversion. In contrast, burning involves permanently removing tokens from circulation, reducing the total supply. Burning can have purposes like deflation, supply reduction, or asset destruction within a blockchain network.
+
+In the Interchain Token Service, incoming token transfers will result in the local minting of new tokens on this blockchain by the Token Manager. Outgoing token transfers will result in the local burning of received tokens.
+
+## Operator
+An operator is a designated role or entity with the authority to perform specific actions or configurations within a blockchain network.
+In Interchain Token Service, the Operator is a role for a Token Manager who can set the `flowLimit` of this manager to limit the influx/outflux of tokens. 
+
+## Standardized
+When a token or smart contract is described as "Standardized" in the blockchain context, it follows recognized industry standards, interfaces, or specifications. This adherence to standards ensures that the token or contract behaves consistently and predictably within the blockchain ecosystem. 
+In the Interchain Token Service, it's a minimal ERC-20 that implements the `IInterchainToken` and `IStandardizedToken` interfaces and can be deployed by the service. 
+
+## Token Manager
```

Comment:
Add link to example one on a blockchain explorer.

Reply:
Or the source code.

Reply:
Updated